### ISSUE TITLE
Build failure on platform CentOS 6 with g++ 5.1

### ISSUE THREAD

**Issue by pdhahn** [fltk/fltk#855]
**Describe the bug**
Build fails on platform CentOS 6 with g++ 5.1.

Here is the relevant output snippet from make (Makefile generated by cmake):
```c++
[ 44%] Building CXX object src/CMakeFiles/fltk_gl.dir/drivers/X11/Fl_X11_Gl_Window_Driver.cxx.o
/vol1/athabasca/apps/fltk/fltk-clone/src/drivers/X11/Fl_X11_Gl_Window_Driver.cxx:417:35: error: ‘int (* glXGetSwapIntervalMESA)()’ redeclared as different kind of symbol
 static MESA_Get_Swap_Iterval_Proc glXGetSwapIntervalMESA = NULL;
                                   ^
In file included from /vol1/athabasca/apps/fltk/fltk-clone/src/drivers/X11/Fl_X11_Gl_Window_Driver.cxx:23:0:
/usr/include/GL/glx.h:398:12: note: previous declaration ‘int glXGetSwapIntervalMESA()’
 extern int glXGetSwapIntervalMESA(void);
            ^
/vol1/athabasca/apps/fltk/fltk-clone/src/drivers/X11/Fl_X11_Gl_Window_Driver.cxx: In function ‘void init_swap_interval()’:
/vol1/athabasca/apps/fltk/fltk-clone/src/drivers/X11/Fl_X11_Gl_Window_Driver.cxx:430:28: error: assignment of function ‘int glXGetSwapIntervalMESA()’
     glXGetSwapIntervalMESA = (MESA_Get_Swap_Iterval_Proc)glXGetProcAddressARB((const GLubyte*)"glXGetSwapInte
                            ^
/vol1/athabasca/apps/fltk/fltk-clone/src/drivers/X11/Fl_X11_Gl_Window_Driver.cxx:430:28: error: cannot convert ‘MESA_Get_Swap_Iterval_Proc {aka int (*)()}’ to ‘int()’ in assignment
make[2]: *** [src/CMakeFiles/fltk_gl.dir/drivers/X11/Fl_X11_Gl_Window_Driver.cxx.o] Error 1
make[1]: *** [src/CMakeFiles/fltk_gl.dir/all] Error 2
```

This appears to be some kind of name collision.

**FLTK Version**
Please complete the following information and delete non-applicable lines:
 - Version: 1.4.0
 - Downloaded and self-built from: git
 - If from Git, branch: master
 - If from Git, commit: 9b33cadbc1199cc4727500f3844fc63cd6a8cc93

**FLTK Configure / Build Options**
I use cmake-gui. I select the Release build. Here are the Options I select:

![image](https://github.com/fltk/fltk/assets/18319324/223651c0-99d4-4e32-b1cd-2b6fc14ee8d4)

**Operating System / Platform:**
Please be as precise as possible, e.g. "Linux: Ubuntu 20.04"
 - OS: Linux
 - OS Version: CentOS 6
 - Processor: Intel Xeons

**Linux/Unix Runtime, if applicable:**
 - X11

**Additional Context**
Yes, the platform I refer to here is old but still useful. I only update FLTK (1.4) about every month or so on it. This build problem did not appear in my last iteration (not sure how long ago that was, but not more than several months).

Importantly, I have not changed anything else (no drivers, nothing GL-related, etc.) on this platform since I last built FLTK successfully.

**Comment by MatthiasWM** [comment]
Thank for posting this issue. I will solve the name conflict ASAP.

**Comment by MatthiasWM** [comment]
PS: yes, this is brand new code on our side.

**Comment by MatthiasWM** [comment]
Could you please verify that the name conflict is solved for 1077d749585dbb0778b8155f35c7c1fa2d0ee601 (current master) and the library compiles again on your machine. Thanks.

**Comment by pdhahn** [comment]
Yes, I just downloaded it and it built fine on my CentOS 6 system. Thank you!
